TombEngine/Objects/TR3/Entity/tr3_shiva.cpp

```diff
@@ -21,16 +22,18 @@ using std::vector;
 
 namespace TEN::Entities::Creatures::TR3
 {
-	constexpr auto SHIVA_GRAB_ATTACK_DAMAGE		= 150;
+	constexpr auto SHIVA_WALK_TURN_RATE_MAX = ANGLE(4.0f);
+	constexpr auto SHIVA_ATTACK_TURN_RATE_MAX = ANGLE(4.0f);
+	constexpr auto SHIVA_DOWNWARD_ATTACK_RANGE = SQUARE(SECTOR(1));
+	constexpr auto SHIVA_GRAB_ATTACK_RANGE = SQUARE(SECTOR(1.25f));
+	constexpr auto SHIVA_GRAB_ATTACK_DAMAGE = 150;
 	constexpr auto SHIVA_DOWNWARD_ATTACK_DAMAGE = 180;
-
-	constexpr auto LARA_ANIM_SHIVA_DEATH = 7;
-
-	#define SHIVA_WALK_TURN_RATE_MAX   ANGLE(4.0f)
-	#define SHIVA_ATTACK_TURN_RATE_MAX ANGLE(4.0f)
+	constexpr auto SHIVA_MESH_SWITCH_TIME = 50000;
+	constexpr auto LARA_ANIM_SHIVA_DEATH = 7; // TODO: move it to LaraExtraAnims enum.
 
 	const auto ShivaBiteLeft  = BiteInfo(Vector3(0.0f, 0.0f, 920.0f), 13);
 	const auto ShivaBiteRight = BiteInfo(Vector3(0.0f, 0.0f, 920.0f), 22);
+	const auto ShivaEffectMeshPosition = Vector3i(0, 0, 0);
```

Comment:
Why do you define const zero vector to pass into `GetJointPosition`? You can just use `Vector3i::Zero` instead.

Reply:
Oops, will remove it.

Reply:
Fixed